File: exercises/small_problems/easy4/short_long.short.rb
def short_long_short(str1, str2)
  str1.size < str2.size ? str1 + str2 + str1 : str2 + str1 + str2
end



p short_long_short('abc', 'defgh')  == "abcdefghabc"
p short_long_short('abcde', 'fgh')  == "fghabcdefgh"
p short_long_short('', 'xyz')  == "xyz"

File: exercises/jsSmallProblems/easy1/shortLongShort.js
const shortLongShort = (string1, string2) => {
  const [short, long] = string1.length < string2.length
    ? [string1, string2] : [string2, string1];

  return short + long + short;
};

shortLongShort('abc', 'defgh'); // "abcdefghabc"
shortLongShort('abcde', 'fgh'); // "fghabcdefgh"
shortLongShort('', 'xyz'); // "xyz"
